### ISSUE TITLE
Regarding the model_file name---"mars-small128.pb".

### ISSUE THREAD

**Issue by naren142** [Qidian213/deep_sort_yolov3#31]
Hi,"mars-small128.pb" model file is trained with the person re-identification dataset right? If yes is it okay to use this model for some other objects on which we want to run the tracking?

**Comment by Qidian213** [comment]
yes ，but its better to train a model on the objects datasets you want to track .
